example/tensorflow/code_template/tensorflow_fashion_mnist.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.keras.datasets import fashion_mnist
+
+(train_x, train_y), (test_x, test_y) = fashion_mnist.load_data()
+
+train_shape = train_x[0].shape
+
+tf.reset_default_graph()
+
+
+# convolution net initializator using low-level API of tensorflow
+def conv_net(x, keep_prob):
+    shape_x = tf.reshape(x, [-1] + list(train_shape) + [1])
+
+    conv1_filter = tf.Variable(tf.truncated_normal(shape=[1, 1, 1, 64], mean=0, stddev=0.08))
+    conv2_filter = tf.Variable(tf.truncated_normal(shape=[1, 1, 64, 128], mean=0, stddev=0.08))
+
+    conv1 = tf.nn.conv2d(shape_x, conv1_filter, strides=[1, 1, 1, 1], padding='SAME')
+    conv1 = tf.nn.relu(conv1)
+    conv1_pool = tf.nn.max_pool(conv1, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='SAME')
+    conv1_bn = tf.layers.batch_normalization(conv1_pool)
+
+    conv2 = tf.nn.conv2d(conv1_bn, conv2_filter, strides=[1, 1, 1, 1], padding='SAME')
+    conv2 = tf.nn.relu(conv2)
+    conv2_pool = tf.nn.max_pool(conv2, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='SAME')
+    conv2_bn = tf.layers.batch_normalization(conv2_pool)
+
+    flat = tf.contrib.layers.flatten(conv2_bn)
+
+    dense = tf.layers.dense(flat, 128, activation=tf.nn.tanh)
+    dense = tf.nn.dropout(dense, keep_prob)
+    dense = tf.layers.batch_normalization(dense)
+
+    out = tf.layers.dense(dense, 10, activation=None)
+    return out
+
+
+def train_neural_network(session, optimizer, keep_probability, feature_batch, label_batch):
```

Comment:
You can rename `session` to `sess` for consistency with rest of the code. Add one liner comment on the purpose of the function.